src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -627,6 +627,12 @@ int SolverInterfaceImpl::getMeshVertexSize(
 {
   PRECICE_TRACE(meshID);
   PRECICE_REQUIRE_MESH_USE(meshID);
+  // In case we access received mesh data: check, if the requested mesh data has already been received.
+  // Otherwise, the function call doesn't make any sense
+  PRECICE_CHECK((_state == State::Initialized) || _accessor->isMeshProvided(meshID), "initialize() has to be called before accessing"
```

Comment:
We should add this precondition to the doc of `getMeshVertexSize`.

Reply:
Good point